OpenGDPR_specification.md

```diff
@@ -0,0 +1,723 @@
+Host: example-processor.com
+Accept: application/json
+```
+
+### 9.2.  Cancellation Response Properties
+For well formed requests, the OpenGDPR service **MUST** respond with HTTP status
+code 202, and the following parameters:
+
+`controller_id`
+
+  **REQUIRED** string indicating the unique identity of the Controller in the
+  Processor's system.
+
+`received_time`
+
+  **REQUIRED** RFC 3339 date string representing the time when the Processor received the cancellation request.
+
+`encoded_request`
+
+  **REQUIRED** Base64 encoding of the entire body of the OpenGDPR request.
+  Controllers **MUST** not log or store this.
+
+`subject_request_id`
+
+  **REQUIRED** UUID v4 string from the originating OpenGDPR request.
+
+`processor_signature`
+
+  **REQUIRED** Base64 encoded signature of the SHA 256 digest of the body of the
+  response.
+
+`api_version`
+
+  **OPTIONAL** Version string representing the desired version of the OpenGDPR
+  API.
+
+
+### 9.3.  Example OpenGDPR Cancellation Response
+
+```http
+HTTP/1.1 202 Accepted
+Content Type: application/json
+X-OpenGDPR-Processor-Domain: example-processor.com
+X-OpenGDPR-Signature:
+kiGlog3PdQx+FQmB8wYwFC1fekbJG7Dm9WdqgmXc9uKkFRSM4uPzylLi7j083461xLZ+mUloo3tpsmyI
+Zpt5eMfgo7ejXPh6lqB4ZgCnN6+1b6Q3NoNcn/+11UOrvmDj772wvg6uIAFzsSVSjMQxRs8LAmHqFO4c
+F2pbuoPuK2diHOixxLj6+t97q0nZM7u3wmgkwF9EHIo3C6G1SI04/odvyY/VdMZgj3H1fLnz+X5rc42/
+wU4974u3iBrKgUnv0fcB4YB+L6Q3GsMbmYzuAbe0HpVA17ud/bVoyQZAkrW2yoSy1x4Ts6XKba6pLifI
+Hf446Bubsf5r7x1kg6Eo7B8zur666NyWOYrglkOzU4IYO8ifJFRZZXazOgk7ggn9obEd78GBc3kjKKZd
+waCrLx7WV5y9TMDCf+2FILOJM/MwTUy1dLZiaFHhGdzld2AjbjK1CfVzyPssch0iQYYtbR49GhumvkYl
+11S4oDfu0c3t/xUCZWg0hoR3XL3B7NjcrlrQinB1KbyTNZccKR0F4Lk9fDgwTVkrAg152UqPyzXxpdzX
+jfkDkSEgAevXQwVJWBNf18bMIEgdH2usF/XauQoyrne7rcMIWBISPgtBPj3mhcrwscjGVsxqJva8KCVC
+KD/4Axmo9DISib5/7A6uczJxQG2Bcrdj++vQqK2succ=
+{
+  "controller_id": "example_controller_id",
+  "subject_request_id": "a7551968-d5d6-44b2-9831-815ac9017798",
+  "received_time": "2018-10-02T15:00:01Z",
+  "encoded_request": "<BASE64 ENCODED REQUEST>",
+  "api_version": "0.1"
+}
+```
+
+## 10.   Best Practices
+
+All Parties **MUST** make best efforts to not throttle during normal operation.
+
+Controllers **MAY** use a sub-processor to manage their OpenGDPR request infrastructure
+to ensure requests are distributed reliably, security signatures are verified and
+thorough logs are kept.
+
+## 11.   Security Considerations
+
+The intention of this specification is to improve data subject privacy by making it
+easier to fulfill their GDPR rights. In doing so, there is a risk to leaking data
+subject identities. Implementers are encouraged to take reasonable measures to
+safeguard each request and it’s encapsulated identities.
+
+## 12.   Conclusions
+
+None.
+
+## 13.   References
+
+[The EU General Data Protection Regulation](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32016R0679&from=EN)
+
+### 13.2. Informative References
```

Comment:
Can remove these last sections (Acknowledgements/Appendix) as they're only present in the standards format